Assets/HoloToolkit/Utilities/Tests/TextToSpeechManagerTest.cs

```diff
@@ -4,21 +4,34 @@
 using UnityEngine.Windows.Speech;
 using HoloToolkit.Unity;
 using System;
+using System.Collections.Generic;
 
 public class TextToSpeechManagerTest : MonoBehaviour
 {
     private GestureRecognizer gestureRecognizer;
     public TextToSpeechManager textToSpeechManager;
+    public List<GameObject> agents = new List<GameObject>();
 
     // Use this for initialization
-    void Start ()
+    void Start()
     {
         // Set up a GestureRecognizer to detect Select gestures.
         gestureRecognizer = new GestureRecognizer();
         gestureRecognizer.TappedEvent += GestureRecognizer_TappedEvent;
         gestureRecognizer.StartCapturingGestures();
 
     }
+    void Update()
+    {
+        // For all registered cubes that can talk, show them in green while they talk, in red otherwise
+        foreach(GameObject agent in agents)
+        {
+            if (agent.GetComponent<TextToSpeechManager>().IsSpeaking())
+                agent.GetComponent<Renderer>().material.color = Color.green;
```

Comment:
NIT: Please follow the curly brace patter to future proof code.
if(condition)
{
}
else
{
}